docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.8"
+services:
+  aiidalab:
+    build:
+      context: .
+      args:
+        - aiidalab_home_version=v21.10.0
+    ports:
+      - ${AIIDALAB_PORT:-8888}:8888
+    environment:
```

Comment:
Can you also add the possibility to modify the default username? For an example, see #214